config/samples/kubeadm_v1alpha1_kubeadmconfig.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: bootstrap.cluster.sigs.k8s.io/v1alpha1
+apiVersion: bootstrap.cluster.sigs.k8s.io/v1alpha2
```

Comment:
nit (for followup): We should probably add `config/samples` to .gitignore and remove these samples since they are not valid configs and just the autogenerated files from kubebuilder.

Reply:
💯 i can do that in this pr